packages/core/src/components/tag-input/_tag-input.scss

```diff
@@ -36,38 +50,24 @@ $ti-icon-padding-large: ($pt-input-height-large - $pt-icon-size-standard) / 2;
   .pt-input-ghost {
     // input fills remaining line
     flex: 1 1 auto;
-    margin-bottom: $ti-padding;
     // essentially a min-width, cuz flex allows it to grow or shrink:
-    width: $pt-grid-size * 10;
-    line-height: $tag-height;
```

Comment:
we hit a small regression here:
![screen recording google chrome](https://user-images.githubusercontent.com/199754/35833053-73bfbe7c-0a84-11e8-8154-aac724028ed3.gif)

`line-height: $pt-grid-size * 2` fixes it -- 20px because there's 5px margin at the top and at the bottom

Reply:
ok sure makes sense 👍